demisto_sdk/utils/utils.py

```diff
@@ -61,3 +64,43 @@ def check_configuration_file(command, args):
 
         except MissingSectionHeaderError:
             pass
+
+
+def retry(
+    times: int = 3,
+    delay: int = 1,
+    exceptions: Union[Tuple[Type[Exception]], Type[Exception]] = Exception,
```

Comment:
is there a way to make this varargs? then can always take a list. Also check that exceptiongroups fit into this (think it should no issue)